src/index.html

```diff
@@ -11,12 +11,113 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans&family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
```

Comment:
Missing BEM parent Block:

Remember to add the parent Block name when declaring a child. There are 2 possible scenarios here:
 1. **logo** is an _Element_ of the _Block_ **header**:
 ```
<a
  class="header__logo"
  href="index.html"
>
```
 2. **logo** is a _Block_, but also child of the **header** _Block_:
  ```
<a
  class="header__logo logo"
  href="index.html"
>
```

Choose as you want, and remember it will alter the class of **logo**'s children